pages/windows/netstat.md

```diff
@@ -0,0 +1,12 @@
+# netstat
+
+> Displays active TCP connections, ports being listening, network adapter statistics, the IP routing table, IPv4 statistics and IPv6 statistics.
```

Comment:
```suggestion
> Displays active TCP connections, ports on which the computer is listening, network adapter statistics, the IP routing table, IPv4 statistics and IPv6 statistics.
```
I guess we can stick with the original description instead of `ports being listening`. Or something like `listened` which I don't think can be used here. What do you think @navarroaxel ?